### DISCUSSION TITLE
Some general feedback from using it today.

### DISCUSSION THREAD

**Discussion by cormacmccarthyireland** [HCL-TECH-SOFTWARE/domino-online-meeting-integration#9] [General]
Just to give you some feedback because I, for one am delighted getting access to this so early in the process.

Firstly- Super work here. 

Here's some minor feedback:

- Doing an "online" install (with Domi and the mail template on the Domino server) worked relatively quickly for me (from Ireland with a Domino 12 beta server based in Finland), the replace design of a mailfile took longer for reference.

- There's a typo on the Installation ("installking") page for DOMI - "Installation will run quickest from a local replica of this database installking to a local replica of the mail template."
 
- The font, look and feel of the meeting credentials documents, could probably align more with either the current mail template or maybe some of the new v.12 look and feel (the new R12 nab, new domino backup and certstore.nsf)?
 
- Maybe eventually a GUI based button for entering a sametime URL would be easier down the line, rather than editing script library (though to be fair, it was straight forward doing that with the instructions)?
 
- I'm used the HCL based app, it seemed to work fine, however I can't really test that properly as the only two services that we use internally in our organisation are Sametime meetings (which has the bug "Sametime not getting token #3" that Paul has already logged), and Zoom, which gets the error mentioned on the demo you gave us "You cannot authorize the app This app cannot be installed outside of the developer's account. Please contact the application developer to get support with installing this application." error, so I can't really test the services beyond that for now.  Looking forward to both of them getting sorted so can test in anger

Just thought you'd appreciate knowing that we have started "kicking the tyres" with it.

(I could get access to customer Teams account, but I'm not going to do that until you tell us you're ready for us to do so.)

Thanks,
Cormac

**Comment by rboccadoro** [comment]
Zoom needs a paid account, just like GoToMeeting. Maybe HCL should put a warning about this in the documentation.
I have tested with a customer Teams account and works like a charm.

**Comment by paulswithers** [comment]
Thanks for the feedback, I've raised an issue for the typo.

The online meeting credentials form should be aligning with the layout and fonts of v12, but we'll review as beta 3 progresses.

There were a few options we considered for Sametime URL - profile document, notes.ini pushed with MarvelClient, shared field etc. Our preference was to have it in the design, because it's easier to troubleshoot - it _should_ be identical for all users, no need to introspect mail database itself. Pushing that into the design from the UI gave a level of risk although I'm sure it would have been possible with the minds we had! We decided editing the script library was a step that was not too much of a barrier to adoption for customers.

**Comment by cormacmccarthyireland** [comment]
Thanks @paulswithers 

@rboccadoro I just tried Zoom from a paid account and get the same results.  I think they mentioned the issue in the demo too?  Zoom need to "approve" the app before it will work.  I might be mistaken here, but that was my understanding.

I've tried the free webex account also this morning and that does work perfectly.  Great to see it working end to end :)
